chapters/3-measurement.tex

```diff
@@ -345,95 +456,102 @@ \subsection{Structuring complex data}
 		https://dimewiki.worldbank.org/Monitoring\_Data}}
-is very important for understanding whether field realities match the research design.
-\index{monitoring data}
-Monitoring data is used to understand if the
-assumptions made during the research design corresponds to what is true in reality.
-The most typical example is to make sure that, in an experimental design,
+is very important for understanding if the
+assumptions made during the research design 
+corresponds to what is true in reality.
+The most typical example is to make sure that, 
+in an experimental design,
 the treatment was implemented according to your treatment assignment.
+While it is always better to monitor all activities,
+it might be to costly. 
+In those cases you can sample a smaller number of critical activities and monitor them.
+This will not be detailed enough to be used as a control in your analysis,
+but it will still give a way to 
+estimate the validity of your research design assumptions.
 
 Treatment implementation is often carried out by partners,
 and field realities may be more complex realities than foreseen during research design.
 Furthermore, the field staff of your partner organization,
 might not be aware that their actions are the implementation of your research.
-Therefore, you must acquire monitoring data that tells you how well the treatment assignment in the field
+Therefore, you must acquire monitoring data that 
+tells you how well the treatment assignment in the field
 corresponds to your intended treatment assignment,
-for nearly all research designs.
+for nearly all experimental research designs.
 
-Another example of a research design where monitoring data is important
+An example of a non-experimental research design
+for which monitoring data also is important
 are regression discontinuity (RD) designs
-where the discontinuity is a cutoff for eligibility of the treatment. 
+where he discontinuity is 
+a cutoff for eligibility of the treatment. 
 For example, 
-let's say your project studies the impact of a program for students that scored under 50\% at a test.
+let's say your project studies the impact of a program
+for students that scored under 50\% at a test.
 We might have the exact results of the tests for all students, 
 and therefore know who should be offered the program, 
 however that is not the same as knowing who attended the program. 
 A teacher might offer the program to someone that scored 51\% at the test,
 and someone that scored 49\% at the might decline to participate in the program.
-We should not pass judgment on a teacher that offers a program to a student
-they think can benefit from it, 
-but if that was not inline with our research assumptions,
-then we need to understand how common that was.
+We need to understand how common this was, 
+and if one case was more common than the other.
 Otherwise the result of our research will not be helpful
 in evaluating the program.
```

Comment:
Suggest to cut

Reply:
cut what? the full RD example?

Reply:
oh, I see now what the lines are referring to. You are suggesting to cut the RD example. I'd be ok with that. But I wanted to make the point that monitoring data is not just for experimental data. @mariaruth gave that feedback on an earlier revision, and I agreed with that, and my solution was to include this example.

Reply:
@bbdaniels to wrap in the above rewrite